builtin/logical/ssh/path_config_ca.go

```diff
@@ -440,9 +433,9 @@ func (b *backend) createManagedKey(ctx context.Context, s logical.Storage, manag
 	}
 
 	entry, err := logical.StorageEntryJSON(caManagedKeyStoragePath, &managedKeyStorageEntry{
-		PublicKey: string(keyInfo.PublicKey().Marshal()),
-		KeyName:   keyInfo.Name,
-		KeyId:     keyInfo.Uuid,
+		PublicKey: string(ssh.MarshalAuthorizedKey(keyInfo.PublicKey())),
```

Comment:
Why was this changed? I looked very quickly but the two functions don't seem to be the same output

Reply:
The old one just ouputs a byte array that doesn't convert well to a string. This one creates the SSH-format public key that we actually want